ckanext/harvest/plugin/__init__.py

```diff
@@ -166,6 +169,120 @@ def after_show(self, context, data_dict):
 
         return data_dict
 
+    # CKAN >= 2.10 hooks
+    def after_dataset_create(self, context, data_dict):
```

Comment:
You are still duplicating the same logic in `after_create()` and `after_dataset_create()`. The idea is that the actual logic is located in the hook that will be supported going forward (`after_dataset_create()`) and that `after_create()` is just a call to it. Check my example in the previous comment:

```python
    # CKAN < 2.10 hooks
    def after_create(self, context, data_dict):
        return self.after_dataset_create(context, data_dict)
```